src/data/wallets/wallet-data.ts

```diff
@@ -2083,6 +2084,63 @@ export const walletsData: WalletData[] = [
     onboard_documentation: "https://clear-wallet.flashsoft.eu/",
     documentation: "https://clear-wallet.flashsoft.eu/docs/",
   },
+  {
+    last_updated: "2024-11-03",
+    name: "Coin98 Super Wallet",
+    image: Coin98Image,
+    twBackgroundColor: "bg-[#F0C43F]",
+    twGradiantBrandColor: "from-[#F0C43F]",
+    url: "https://coin98.com/",
+    active_development_team: true,
+    languages_supported: [
+      "en",
+      "vi",
+      "es",
+      "fr",
+      "zh",
+      "id",
+      "ja",
+      "ko",
+      "pt",
+      "ru",
+      "th",
+      "tr",
+    ],
+    twitter: "https://x.com/coin98_wallet",
+    discord: "https://discord.com/invite/K6N7ry5K8d",
+    reddit: "",
+    telegram: "https://t.me/+rPevQX9QSpExMzE1",
+    ios: true,
+    android: true,
+    linux: false,
+    windows: false,
+    macOS: false,
+    firefox: false,
+    chromium: true,
+    hardware: false,
+    open_source: false,
+    repo_url: "",
+    non_custodial: true,
+    security_audit: ["Certik"],
+    scam_protection: false,
+    hardware_support: true,
+    rpc_importing: true,
+    nft_support: true,
+    connect_to_dapps: true,
+    staking: true,
+    swaps: true,
+    layer_2: true,
+    gas_fee_customization: true,
+    ens_support: true,
+    erc_20_support: true,
+    buy_crypto: true,
+    withdraw_crypto: false,
+    multisig: false,
+    social_recovery: false,
+    onboard_documentation: "https://docs.coin98.com/",
+    documentation: "https://docs.coin98.com/",
+    new_to_crypto: true,
```

Comment:
@khiemsoft please remove this field per @konopkja's comment. I don't have write access to this PR or I'd do it myself :)

Could you also clear merge conflicts and I'll get this merged?

Thanks 👍 

```suggestion
```

Reply:
@minimalsm Already submitted your fixed suggestion; thanks for your review.